runtime/dart_controller.h

```diff
@@ -19,6 +19,7 @@ class DartController {
   DartController();
   ~DartController();
 
+  void RunFromKernel(const uint8_t* buffer, size_t size);
```

Comment:
Same comment as above. Happy to tackle this in a subsequent PR. Is `RunFromSource` the pattern I should follow?